tags/tags-impl/impl/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>tags-impl</name>
+  <groupId>org.sakaiproject.tags</groupId>
+  <artifactId>tags-impl</artifactId>
+  
+  <packaging>jar</packaging>
+  
+  <parent>
+    <artifactId>tags-base</artifactId>
+    <groupId>org.sakaiproject.tags</groupId>
+    <version>12-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <properties>
+    <deploy.target/>
+  </properties>
+
+
+  <build>
+    <resources>
+      <resource>
+        <directory>${basedir}/src/resources</directory>
+      </resource>
+    </resources>
+  </build>
+
+  <dependencies>
+    
+    <dependency>
+      <groupId>com.googlecode.json-simple</groupId>
+      <artifactId>json-simple</artifactId>
+      <version>1.1.1</version>
```

Comment:
About the i18n, this project is using the same tool structure than PAsystem, one of the latests tools to be added to the sakai code:
pasystem/pasystem-api/api/src/java/org/sakaiproject/pasystem/api/I18n.java
This was a class in that project that is used later by the service servlet so I didn't changed that and just maintained the same way that the PA tool was working supposing that if that tool was added to Sakai 11 it should be fine.

About the poms, yeah, I agree, I will review them and define versions in the master.